.buildkite/pull-requests.json

```diff
@@ -95,6 +95,38 @@
             "skip_target_branches": [ ],
             "skip_ci_on_only_changed": [ ],
             "always_require_ci_on_changed": [ "^deploy/kubernetes/.*", ".buildkite/deploy/kubernetes/.*", "^libbeat/docs/version.asciidoc"]
+        },
+        {
+            "enabled": true,
+            "pipelineSlug": "beats-libbeat",
+            "allow_org_users": true,
+            "allowed_repo_permissions": ["admin", "write"],
+            "allowed_list": [ ],
+            "set_commit_status": true,
+            "build_on_commit": true,
+            "build_on_comment": true,
+            "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:libbeat))|^/test libbeat$",
+            "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:libbeat))|^/test libbeat$",
+            "skip_ci_labels": [  ],
+            "skip_target_branches": [ ],
+            "skip_ci_on_only_changed": [ ],
+            "always_require_ci_on_changed": [  ]
+        },
+        {
+            "enabled": true,
+            "pipelineSlug": "beats-packetbeat",
+            "allow_org_users": true,
+            "allowed_repo_permissions": ["admin", "write"],
+            "allowed_list": [ ],
+            "set_commit_status": true,
+            "build_on_commit": true,
+            "build_on_comment": true,
+            "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:packetbeat))|^/test packetbeat$",
+            "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:packetbeat))|^/test packetbeat$",
+            "skip_ci_labels": [  ],
+            "skip_target_branches": [ ],
+            "skip_ci_on_only_changed": [ ],
+            "always_require_ci_on_changed": [ "^packetbeat/.*", ".buildkite/packetbeat/.*"]
```

Comment:
Does it mean it will run when files are under those folders? If so, I see it should also include https://github.com/elastic/beats/blob/74945355f9435e61cec4715057d6140dc4a28ab7/packetbeat/Jenkinsfile.yml#L3-L6.

`@ci` and `@oss` can be found in:
- https://github.com/elastic/beats/blob/74945355f9435e61cec4715057d6140dc4a28ab7/Jenkinsfile.yml#L23-L38


Likely `@ci` when using Buildkite will imply any changes in the scripts/pipelines for Buildkite.

Reply:
If I'm not mistaken, the meaning of the "^packetbeat/.*" is everything started with "packetbeat/". It means all files and folders inside the `packetbeat` folder are covered by this regex.
Added the values of the `oss`.
Values of the `ci` are skipped because it's Buildkite migration.